BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/DocumentPages/DocumentPagesViewController.swift

```diff
@@ -101,6 +102,40 @@ final class DocumentPagesViewController: UIViewController {
         showSkontoDetailsInFooter()
     }
 
+    func setError(errorType: ErrorType, buttonAction: @escaping () -> Void) {
+        let errorView = DocumentPagesErrorView(
+            errorType: errorType,
+            buttonTitle: NSLocalizedStringPreferredGiniBankFormat(
+                "ginibank.transactionDocs.preview.error.action",
+                comment: "Try again"),
+            buttonAction: {
+                buttonAction()
+                print("try again pressed")
+                self.removeErrorView()
+            }
+        )
+
+        view.addSubview(errorView)
+
+        errorView.translatesAutoresizingMaskIntoConstraints = false
+
+        NSLayoutConstraint.activate([
+            errorView.topAnchor.constraint(equalTo: navigationBar.bottomAnchor),
+            errorView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            errorView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            errorView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        ])
+
+        self.errorView = errorView
+    }
+
+    func removeErrorView() {
+        if let errorView = self.errorView {
```

Comment:
[suggestion]: Maybe we can do like this:
```
      guard let errorView = errorView else { return }
        errorView.removeFromSuperview()
        self.errorView = nil
        ```
        it feels complete with `guard`